lab_01_transformations/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ add_executable(${PROJECT_NAME}
   )
 
```

Comment:
Får ikke lagt til kommentar på linje 7, så legger den her:
For MSVC måtte jeg spesifisere at jeg ønsker å bygge dynamisk, altså måtte jeg utvide `compiler_options` med `/MD`.